src/validation.js

```diff
@@ -0,0 +1,45 @@
+const validation = {
+  /**
+   *
+   * @param {Array} inputArr
+   * @returns
+   */
+  isLengthBelowFive: function isLengthBelowFive(inputArr) {
+    return inputArr.every((input) => input.length < 6);
+  },
+  /**
+   *
+   * @param {Array} inputArr
+   * @returns {Boolean}
+   */
+  hasSpace: function hasSpace(inputArr) {
+    return inputArr.some((input) => /\s/g.test(input));
+  },
+  /**
+   *
+   * @param {Array} inputArr
```

Comment:
```suggestion
   * @param {Array<string>} inputArr
```

Array 같은 경우 제네릭을 지정해주면 타입을 조금 더 구체적으로 설정할 수 있어요! 배열안에 있는 값들이 어떤 타입인지 함께 정의되면 좋을 거 같아요